constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/utils/ArcCurveFit.java

```diff
@@ -275,6 +281,15 @@ public ArcCurveFit(int[] arcModes, double[] time, double[][] y) {
                     break;
                 case ARC_START_LINEAR:
                     mode = START_LINEAR;
+                    break;
+                case ARC_UP:
+                    Utils.log("Arc ARC_UP");
```

Comment:
Remove or wrap in something like `if(DEBUG)`

Reply:
removed